backends/ref/ceed-ref-operator.c

```diff
@@ -76,14 +76,16 @@ static int CeedOperatorApply_Ref(CeedOperator op, CeedVector qdata,
     CeedChk(ierr);
   }
   ierr = CeedVectorRestoreArray(etmp, &Eu); CeedChk(ierr);
+  ierr = CeedVectorRestoreArray(qdata, (CeedScalar**)&qd); CeedChk(ierr);
```

Comment:
How did this find its way into this commit (which claims to be adding the template backend)?

Reply:
I added it to not undo your commit since the merge didn't look like it would preserve it.

Reply:
Hmm, it should (at least using the command-line git). Since splitting the file in independent from the template backend, it could go in a separate PR, preferably on top of current 'master'. Either way, it doesn't make sense in this commit.